Fixed: PlayStore showing deep linking issue for URLs schema.

MohitMaliFtechiz opened the pull request:
Fixes #3808 

* Removed the URL type deep link since we are mainly using deep links for opening zim files in the application, and URLs type deep links are for downloading the content not for opening the ZIM files so we are removing these deep links from our application.
* Now user can normally download the zim files from (outside Kiwix) like other zim files.

kelson42 commented:
I hardly undetstand why it does not open the welcome page indeed!!!

kelson42 commented:
I hardly understand why it does not open the welcome page indeed!! Pretty sure all of this was working fine. Please create complete test set for this.

MohitMaliFtechiz commented:
> I hardly understand why it does not open the welcome page of the zim - the user just clixked on - indeed!! Pretty sure all of this was working fine. Please create complete test set for this.

@kelson42 I think here is some confusion, The user clicks on the URL not on the ZIM file.

> Pretty sure all of this was working fine.

Yes, we are handling the scenario where the user clicks on the zim file in his storage. The Zim file will open in the reader if a user clicks on the zim file.

> I hardly understand why it does not open the welcome page of the zim -

Here, it is a link to the Zim file, not the path or URI of the zim file so that we can open the zim file in the reader. The deep link of URL types was probably introduced for opening the download screen but the handling is missing for this schema type.

However, if you want this scenario if a user clicks on the link then open the corresponding zim file reader and open the welcome page of zim file. so for this, we can make modifications to our code. 

But it has some edge cases:
* Since this is a URL, not the URI or path so we can't directly open the zim file in the reader, and we are saving the `zimFilePath` in our db with the `fileName` that we had put when the user refreshed the zim file data on LocalLibraryScreen. But if the user has not refreshed the data and tries to open the zim file by URL then it will fail to load the zim file.
* If the user tries to open a link e.g. https://download.kiwix.org/zim/wikipedia.zim and the related zim file does not exist in the db then also it will fail to load the zim file.
* Also If the user renames the file then it will not open via url.

kelson42 commented:
@MohitMaliFtechiz If someone clicks on an http link leading to a ZIM (outside Kiwix) then nothing special should happen. The file should just be downloaded like any other.

kelson42 commented:
What should we do with this Pr?

MohitMaliFtechiz commented:
@kelson42 I have removed the URL type deep link from the application, and updated the PR description also, to fix the deep link issue reported by the plasytore we need to merge this PR.
